app/config/collections2.php

```diff
@@ -1720,6 +1731,13 @@
                 'lengths' => [Database::LENGTH_KEY],
                 'orders' => [Database::ORDER_ASC],
             ],
+            [
+                '$id' => '_fulltext_search',
+                'type' => Database::INDEX_FULLTEXT,
+                'attributes' => ['search'],
+                'lengths' => [1024],
```

Comment:
I think the index length should match the attribute length
```suggestion
                'lengths' => [16384],
```